src/modules/src/stabilizer.c

```diff
@@ -368,10 +350,6 @@ PARAM_ADD_CORE(PARAM_UINT8, estimator, &estimatorType)
  * @brief Controller type Auto select(0), PID(1), Mellinger(2), INDI(3), Brescianini(4) (Default: 0)
  */
 PARAM_ADD_CORE(PARAM_UINT8, controller, &controllerType)
-/**
- * @brief If set to nonzero will turn off power
- */
-PARAM_ADD_CORE(PARAM_UINT8, stop, &emergencyStop)
```

Comment:
We should probably not remove this since this is a core parameter. Let's follow the deprecation manual https://test-www.bitcraze.io/development/development-overview/#the-update-process-and-deprecations

Reply:
moved to another file